### PR TITLE
Investigate and address User API failures #5799

### PR DESCRIPTION
**Pull request by sergiogcx**
Closes https://github.com/cityofaustin/atd-data-tech/issues/5799

** This is mostly a UI change only, you can test in Netlify, moped-test and locally.

There are API changes that are meant to make sure the user name stays as lowercase in AWS.

Also, there was an upgrade for the DataGrid component where a column's className can now be customized.

Live Demo:
https://deploy-preview-324--atd-moped-main.netlify.app/moped/staff

### CODE REVIEW

**Comment by sergiogcx** [comment]
The email address is now presented without any formatting:

File: moped-api/claims.py [hunk 1]
```diff
@@ -25,6 +25,18 @@
...
+    """
+    Attempts to lower case a user email address
+    :param user_email: The user email address in question
+    :return: 
+    """
+    try:
+        return str(user_email).lower()
+    except:
+        return user_email
+
```
Comment (chiaberry) [inline moped-api/claims.py:38]: 🎊

File: moped-editor/src/views/staff/StaffListViewQueryConf.js [hunk 2]
```diff
@@ -140,7 +141,9 @@ export const StaffListViewQueryConf = {
     },
   },
   order_by: {},
-  where: {},
+  where: {
+    status_id: "_eq: 1"
```
Comment (chiaberry) [inline moped-editor/src/views/staff/StaffListViewQueryConf.js:145]: if we are going to only get the active users, do we still need to show the column `status_id`?

**Comment by chiaberry** [comment]
how can I test the user api failures? is there a way to provoke the error message?

File: moped-editor/src/views/staff/StaffListViewQueryConf.js (hunk 2 above)
Comment (sergiogcx) [reply moped-editor/src/views/staff/StaffListViewQueryConf.js:145]: We don't, good call! 👍

**Comment by sergiogcx** [comment]
> how can I test the user api failures? is there a way to provoke the error message?

Create a dummy user (don't worry it will fail), with a bad password without numbers and/or special characters.

File: moped-editor/src/views/staff/StaffListViewQueryConf.js (hunk 2 above)
Comment (mateoclarke) [reply moped-editor/src/views/staff/StaffListViewQueryConf.js:139]: I agree with Chia. Doesn't seem necessary

Comment (sergiogcx) [reply moped-editor/src/views/staff/StaffListViewQueryConf.js:139]: It's deleted now :)

**Comment by amenity** [comment]
Thanks, @sergiogcx! This looks great.

**Review by amenity — APPROVED** [review]

**Pull request merged**